staging_vespalib/src/tests/benchmark/testbase.cpp

```diff
@@ -165,12 +166,12 @@ size_t ReturnByValueMultiVectorString::onRun()
 
 size_t ClockSystem::onRun()
 {
-    fastos::UTCTimeStamp start(fastos::ClockSystem::now());
-    fastos::UTCTimeStamp end(start);
+    vespalib::system_time start(vespalib::system_clock::now());
+    vespalib::system_time end(start);
     for (size_t i=0; i < 1000; i++) {
-        end = fastos::ClockSystem::now();
+        end = vespalib::system_clock::now();
     }
-    return (start - end).ns();
+    return (start - end).count();
```

Comment:
Consider explicitly `duration_cast`ing to nanoseconds before `count()`ing since it's not technically defined what the underlying resolution of the system clock is

Reply:
Fixed